docs/FAQ.md

```diff
@@ -1,91 +1,106 @@
+editor, you might need to give it additional permissions to execute it.
 
 If you receive such a permission error on macOS, run:
 
-`chmod -R 755 *.app` 
+```shell
+chmod -R 755 *.app
+```
 
 or on Linux:
 
-`chmod -R 755 *.x86_64` 
+```shell
+chmod -R 755 *.x86_64
+```
 
-On Windows, you can find 
+On Windows, you can find
 [instructions](https://technet.microsoft.com/en-us/library/cc754344(v=ws.11).aspx).
 
-### Environment Connection Timeout
-
-If you are able to launch the environment from `UnityEnvironment` but 
-then receive a timeout error, there may be a number of possible causes.
- * _Cause_: There may be no Brains in your environment which are set 
- to `External`.  In this case, the environment will not attempt to 
- communicate with python. _Solution_: Set the Brains(s) you wish to 
- externally control through the Python API to `External` from the 
- Unity Editor, and rebuild the environment.
- * _Cause_: On OSX, the firewall may be preventing communication with 
- the environment. _Solution_: Add the built environment binary to the 
- list of exceptions on the firewall by following 
- [instructions](https://support.apple.com/en-us/HT201642). 
- * _Cause_: An error happened in the Unity Environment preventing 
- communication. _Solution_: Look into the 
- [log files](https://docs.unity3d.com/Manual/LogFiles.html) 
- generated by the Unity Environment to figure what error happened. 
-
-### Communication port {} still in use
-
-If you receive an exception `"Couldn't launch new environment because 
-communication port {} is still in use. "`, you can change the worker 
-number in the Python script when calling 
+## Environment Connection Timeout
+
+If you are able to launch the environment from `UnityEnvironment` but then
+receive a timeout error, there may be a number of possible causes.
+
+* _Cause_: There may be no Brains in your environment which are set to
+  `External`.  In this case, the environment will not attempt to communicate
+  with python. _Solution_: Set the Brains(s) you wish to externally control
+  through the Python API to `External` from the Unity Editor, and rebuild the
+  environment.
+* _Cause_: On OSX, the firewall may be preventing communication with the
+  environment. _Solution_: Add the built environment binary to the list of
+  exceptions on the firewall by following
+  [instructions](https://support.apple.com/en-us/HT201642).
+* _Cause_: An error happened in the Unity Environment preventing communication.
+  _Solution_: Look into the [log
+  files](https://docs.unity3d.com/Manual/LogFiles.html) generated by the Unity
+  Environment to figure what error happened.
+
+## Communication port {} still in use
+
+If you receive an exception `"Couldn't launch new environment because
+communication port {} is still in use. "`, you can change the worker number in
+the Python script when calling
 
 `UnityEnvironment(file_name=filename, worker_id=X)`
```

Comment:
This is python code

Reply:
Cool, changed the docs to highlight it as Python.